### PR TITLE
Tabstrip design revision

### PR DESCRIPTION
**Pull request by silviyaboteva**
Closes: https://github.com/telerik/kendo-themes-private/issues/267

### CODE REVIEW

**Comment by inikolova** [comment]
@silviyaboteva great work :) Please find my suggestions below:
- [ ] In my opinion, tabstrip alignment options should be presented as a test, but not as a template;
- [ ] Add a test for scrollableButtonsPosition "around";

File: packages/html/src/tabstrip/tabstrip.spec.tsx [hunk 1]
```diff
@@ -63,11 +77,23 @@ export const TabStrip = (
...
             {position === "bottom" && children}
             <TabStripItemsWrapper orientation={orientationMap[position]}>
-                {scrollable && <Button fillMode="flat" rounded={null} size={null} icon={`caret-alt-${caretMap[position]["prev"]}`} className="k-tabstrip-prev"></Button>}
-                <TabStripItems>
+                {scrollable &&
+                    (scrollableButtonsPosition === 'around' || scrollableButtonsPosition === 'start') &&
+                    <Button fillMode="flat" rounded={null} size={"medium"} icon={`caret-alt-${caretMap[position]["prev"]}`} className="k-tabstrip-prev"></Button>
+                }
+                {scrollable && scrollableButtonsPosition === 'start' &&
+                    <Button fillMode="flat" rounded={null} size={null} icon={`caret-alt-${caretMap[position]["next"]}`} className="k-tabstrip-next"></Button>
```
[the remark this answers is not in the document]
Comment (silviyaboteva) [reply packages/html/src/tabstrip/tabstrip.spec.tsx:85]: I missed that. Will be set to 'medium'. Thanks!

File: packages/html/src/tabstrip/tests/tabstrip-closable.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import { TabStripBottom, TabStripContent, TabStripItem, TabStripLeft, TabStripNormal, TabStripRight } from '..';
+
+export default () => (
+    <>
+        <div id="test-area" className="k-d-grid k-grid-cols-2">
+            <section>
+                <TabStripNormal
```
Comment (inikolova) [inline packages/html/src/tabstrip/tests/tabstrip-closable.tsx:7]: As this test demonstrates scrollable tabs, maybe we could use TabStripClosable instead?

**Comment by inikolova** [comment]
One more thing, Angular TabStrip has a tabAlignment option for tabs alignment already implemented - in my opinion it is a good practice to use the same property names (where possible) across all suites. Hence my suggestion is to rename the alignment prop to tabAlignment.

**Comment by silviyaboteva** [comment]
> One more thing, Angular TabStrip has a tabAlignment option for tabs alignment already implemented - in my opinion it is a good practice to use the same property names (where possible) across all suites. Hence my suggestion is to rename the alignment prop to tabAlignment (or tabsAlignment).

Done! Thanks! <3

File: packages/default/scss/tabstrip/_layout.scss [hunk 3]
```diff
@@ -71,6 +71,13 @@
             align-items: stretch;
             justify-items: stretch;
             outline: 0;
+
+            // Active state
+            &:active,
+            &.k-active,
+            &.k-selected {
+                font-weight: var( --kendo-tabstrip-item-selected-font-weight, #{$kendo-tabstrip-item-selected-font-weight} );
```
Comment (inikolova) [inline packages/default/scss/tabstrip/_layout.scss:79]: The font weight here should be set in the following way:
font-weight: $kendo-tabstrip-item-selected-font-weight;

**Comment by inikolova** [comment]
One final thing which I noticed and believe we should consider. The tabbed column menu in Grid uses Tabstrip with icons only (without text). The rendering there suggest that the icon is inside the ".k-link-text" wrapper:
https://github.com/telerik/kendo-themes/blob/tabstrip-revision/tests/grid/grid-column-menu-tabbed.html#L38
The current rendering of the Tabstrip component suggests that the icon and the "k-link-text" elements are siblings
https://github.com/telerik/kendo-themes/blob/tabstrip-revision/tests/tabstrip/tabstrip-icons.html#L29
In my opinion a change in the rendering is needed and both components should have the same HTML

**Comment by silviyaboteva** [comment]
@inikolova Thank you for this catch! 

✅ As we now have the option of passing text only or icon only, or both, I'll go ahead and update the Grid Column menu test to use the new `icon` prop. It will result in correct rendering as the icon element should not be rendered as a child of `k-link-text` but as a sibling prior or after the text (if `iconPosition` is passed). This is synced with @zhpenkov and @Juveniel 

✅ Also, from the sync with @Juveniel, I have added `actions` prop which allows tabstrip item to have more than one action (for example in Dock Manager where there are `pin` and `more` actions). The actions will be rendered inside `k-item-actions` element **including the remove button**.

@zhpenkov when you have the chance, please check it :)

File: tests/_output/material/spreadsheet/spreadsheet.png
[the remark this answers is not in the document]
Comment (silviyaboteva) [reply tests/_output/material/spreadsheet/spreadsheet.png:1]: we need to align them in the center. Thanks!

File: tests/spreadsheet/spreadsheet.html [hunk 4]
```diff
@@ -719,78 +719,102 @@
                         </svg>
                     </span>
                 </button>
-                <div class="k-tabstrip k-spreadsheet-sheets k-tabstrip-bottom k-tabstrip-scrollable">
-                    <div class="k-tabstrip-items-wrapper k-vstack">
-                        <ul class="k-tabstrip-items k-reset">
+                <div class="k-tabstrip k-spreadsheet-sheets k-tabstrip-bottom k-tabstrip-scrollable k-header">
+                    <div class="k-tabstrip-items-wrapper k-hstack">
```
[the remark this answers is not in the document]
Comment (silviyaboteva) [reply tests/spreadsheet/spreadsheet.html:723]: I have added `header` property to false to remove the `k-header` class. The k-hstack or k-vstack in this concrete scenario doesn't produce any visual changes.

File: tests/_output/classic/dock-manager/dock-manager.png
[the remark this answers is not in the document]
Comment (silviyaboteva) [reply tests/_output/classic/dock-manager/dock-manager.png:1]: Previously, the test passed the action buttons themeColor="primary". However, after the design revision, the Default (and Classic theme as it follows the Default) should have reversed colors of active/non-active tabs. That said, the active tab should be with primary (link) color, and the rest non-active tabs should be with component text color.

That being said, the Default, Bootstrap, Material are ok. Fluent, now is also following the designs (https://www.figma.com/design/gWUIux95BHxemqWwOw9ZwB/Default-Theme---Internal-Usage?node-id=11011-431592&node-type=FRAME&t=6bO9FGmuyjzyLT6J-0). The tab content is with component text color, including the text, icons, actions. The active item differs to others by bottom primary border only. 

In Classic theme, the color of the active tab should be passed to the tab item itself (will address that). Non-active stays as the image showcases.

File: tests/_output/bootstrap/tabstrip/tabstrip-actions.png
[the remark this answers is not in the document]
Comment (silviyaboteva) [reply tests/_output/bootstrap/tabstrip/tabstrip-actions.png:1]: Yes, of course! Tanks for noticing this!

**Pull request merged**